src/main/java/org/generationcp/middleware/operation/saver/GeolocationPropertySaver.java

```diff
@@ -26,9 +26,6 @@ public void saveFieldmapProperties(final List<FieldMapInfo> infos) throws Middle
 					int locationId = trial.getInstanceId();
 					if (trial.getInstanceId() != null && trial.getInstanceId().intValue() == 1) {
 						locationId = this.getExperimentModelSaver().moveStudyToNewGeolocation(info.getFieldbookId());
-					}
```

Comment:
Why did you remove null checking for trial getLocationId?

Reply:
Hi @abatac, will revert checking if location is not null before saving